### ISSUE TITLE
Cloudwatch plugin not working after upgrading from 9.4.3 to 10.3.3

### ISSUE THREAD

**Issue by mrachell** [grafana/grafana#83330]
### What happened?

Upgraded Grafana 9.4.3 to 10.3.3. Existing dashboards using the core Cloudwatch plugin (for AWS) stopped working. So I deleted and re-created the Cloudwatch datasource and edited an existing dashboard's panel. I select the new region, log groups, and keep the default query. Then I choose Inspect - Panel JSON - Query. The response includes the error "An error occurred within the plugin". The json looks like this:

Object
  traceId: ...
  request: Object
    url: "api/ds/query?ds_type=cloudwatch&requestId=Q127"
    method: "POST"
    ....
  response: Object
    message: "An error occurred within the plugin"
    messageId: "plugin.downstreamError"
    statusCode: 500

The Grafana logs show this error (had to type it in, so this isn't exact): 

t=2024-02-23T11:11:29.... level=error msg="Internal server error" error="[plugin.downstreamError]" client: failed to query data: failed to execute log action with subtype: StartQuery: MalformedQueryException: token Recognition error at '\\' (Service: AWSLogs: Status Code: 400; Error Code: MalformedQueryException; Request ID: xxxxx; Proxy: null)\n{\n RespMetadata: {\n    StatusCode: 400,\n    RequestID: \"xxxx\"\n    },\n Message_: \"token recognition error at: '\\\\' (Service: AWSLogs; Status Code: 400; Error Code....

### What did you expect to happen?

The query to hit Cloudwatch Logs, execute the query, and return results. I can run this same query directly on the server using the aws cli since the server has an instance role with cloudwatch logs access. 

### Did this work before?

Yes, this worked on Grafana 9.4.3.

### How do we reproduce it?

1. Install Grafana 9.4.3
2. Configure Cloudwatch plugin
3. Create dashboard with a panel to use the Cloudwatch plugin. 
4.  Keep the default query. Go to Inspect - Panel JSON - Query - Refresh
5.  You will get the same error "An error occurred within the plugin"

### Environment (with versions)?

Grafana: 10.3.3
OS: Cent0s 8
Browser: Chrome, Firefox

### Grafana platform?

Docker

### Datasource(s)?

Cloudwatch core plugin that came with Grafana 10.3.3

**Comment by jangaraj** [comment]
Create panels from the scratch and it should be working.

**Comment by mrachell** [comment]
Thank you for the comment. I get the same error "An error occurred within the plugin", "plugin.downstreamError" when I add a new panel to an existing dashboard. I also get that error when I start at Home > Connections > Data sources > cloudwatch. Both Build a Dashboard and Explore options give the same error after I setup the region, log groups, and query. This is telling me it's the datasource / connection that is having a problem, not the dashboard or panel.

**Comment by jangaraj** [comment]
Check Grafana server logs (increase log level eventually), pls.

**Comment by mrachell** [comment]
I modified the panel json a bit, and I was able to get a new error which make it seem like this issue is related to this: https://community.grafana.com/t/grafana-cloudwatch-loggroups-query-error/87779/3. I can edit the panel json to remove the arn prefix (and :* at the end) of the arn: property, and the panel runs. I probably caused part of this problem by deleting the old/previous cloudwatch datasource. Since that's referenced in each panel, I have to copy the query out of each panel's JSON, then edit the log groups to delete the ARN. But I still see that sometimes when I save the panel or the dashboard, it seems like the cloudwatch plugin adds the ARN back to the log group - so I have to re-delete it. There's definitely something wrong with the cloudwatch plugin adding the ARN back to the logGroups: property. I may create another issue just for that.
